docker/internal-auth/compose.yaml

```diff
@@ -15,6 +15,8 @@ services:
       service: teams-api-common
 
   teams-app:
+    environment:
+      FEATURE_FLAG_ENABLE_INVITATIONS: false
```

Comment:
In internal mode, hard code disable by setting `FEATURE_FLAG_ENABLE_INVITATIONS` to `false`, even if user sets to true in `.env`.

Added to teams-app as it should be set here in addition to teams-api.